test/Acceptance.md

```diff
@@ -147,3 +148,24 @@ curl -X GET http://localhost:8090/v2/wallets/1b0aa24994b4181e79116c131510f2abf6c
   },
 
 ```
+
+## Metadata sync
+
+- A sync script is setup to run hourly. This script pulls the current GitHub repository state into the Postgres database.
+
+- To test that this script is working correctly, we need to:
+
+  1. Disable the metadata-registry-testnet webhook
+    - Navigate to https://github.com/input-output-hk/metadata-registry-testnet/settings/hooks/279167325
```

Comment:
Could I get access to the setting, so I can execute this test in future... (it gives me 404 atm)

Reply:
Hi @piotr-iohk - I have made your individual user account be an Admin, and also the `@qa` group. Please try again - I just checked and the URL exists.